footbot/data/utils.py

```diff
@@ -131,3 +150,32 @@ def check_next_event_deadlinetime():
     deadlinetime = datetime.datetime.strptime(deadlinetime_str, "%Y-%m-%dT%H:%M:%SZ")
 
     return deadlinetime < datetime.datetime.now() + datetime.timedelta(hours=24)
+
+
+def get_current_event(bootstrap_data=None):
```

Comment:
nice, we do this a lot..  another occurrence that could be replaced-

https://github.com/thomas-woodruff/footbot/blob/master/footbot/data/entry_data.py#L25

Reply:
I'm going to drop this code altogether I think, as it's not being used.